tests/nlp/checks/train_test_validation/property_drift_test.py

```diff
@@ -108,9 +107,9 @@ def test_without_drift(self, small_wikiann_train_test_text_data):
         assert_that(result.value, has_entries({
             'Text Length': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
             '% Special Characters': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
-            'Sentiment': has_entries({'Drift score': 0.0, 'Method': 'Kolmogorov-Smirnov'}),
+            'Sentiment': has_entries({'Drift score': None, 'Method': None}),
```

Comment:
why has this changed to None? Too many samples that are not in english in wikiann?

Reply:
> Too many samples that are not in english in wikiann?

yeap, many samples were not identified as english, and as a result `not enough samples` for drift calculation

Reply:
mm note to self to try to improve that...